release-notes/MediaOps/MediaOps_1.4.0.md

```diff
@@ -43,6 +43,10 @@ In Resource Studio, several exception messages have been improved to be more cle
 
 In Resource Studio, an icon has been added that indicates whether a capability on a resource is provided by a resource pool.
 
+#### Scheduling: Client time visualized on job and resource timeline [ID 43639]
+
+Though by default the time zone of the client machine is displayed in the Scheduling app, this can be customized through user settings. Therefore, to ensure that operators have good visibility on the current time used in the Scheduling application, a clock has been added to both the *Job View* and *Resource View* pages.
```

Comment:
... time zone of the client machine is applied in the Scheduling app, ...

Reply:
Before it was not displayed, with this change it will be displayed to ensure operators know what the time is according to the application.